google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -365,10 +365,15 @@ class Page:
     """Represents a wrapped documentai.Document.Page .
 
     Attributes:
+        shard_index (int):
+            Required. The index of the `google.cloud.documentai.Document` containing
+            the `google.cloud.documentai.Document.Page`.
         documentai_page (google.cloud.documentai.Document.Page):
-            Required. The original google.cloud.documentai.Document.Page object.
-        text: (str):
-            Required. The full text of the Document containing the Page.
+            Required. The original `google.cloud.documentai.Document.Page` object.
+        text (str):
+            Required. The full text of the `Document` containing the `Page`.
+        page_number (int):
+            Required. The page number of the `Page`.
```

Comment:
Maybe add to the docstring that `page_number` is always page index+1.  But then how is this field used?  Wouldn't `page_index` be more useful?

(In `samples/snippets/table_sample.py` below, for example, the name `page_number` indeed refers to page index.)

Reply:
Page number is populated by document ai directly, and it functions as a 1-based index for the page in the document.

I could also add methods to get the original pages/entities by their unique identifiers (page_number and id respectively)